app/views/layouts/ats/_navbar.html.slim

```diff
@@ -12,28 +16,22 @@ nav.navbar.navbar-expand-lg.align-items-center.fixed-top.header.gap-2
     #navbarNav.collapse.navbar-collapse.gap-lg-2.order-5.order-lg-1
       ul.navbar-nav.text-nowrap.gap-1.gap-lg-0.mb-1.mb-lg-0
         - if allowed_to?(:index?, with: ATS::DashboardPolicy)
```

Comment:
Since these buttons all use the same policy condition, why not combine them under one?

Reply:
oh i see, policy are different
ATS::TaskPolicy ATS::CandidatePolicy etc, I'l keep them